rasa/shared/core/trackers.py

```diff
@@ -703,12 +703,7 @@ def export_stories(
         Returns:
             The dumped tracker as a string.
         """
-
-        # TODO: we need to revisit all usages of this, the caller needs to specify
-        #       the format. this likely points to areas where we are not properly
-        #       handling markdown vs yaml
         story = self.as_story(include_source)
```

Comment:
wait, won't this dump in MD format?

Reply:
It doesn't seem so. 
We only use `story.story_steps` and then we just provide this steps for a writer (which is `YAMLStoryWriter`) so it does work as expected.